vizro-core/src/vizro/static/css/typography.css

```diff
@@ -59,6 +59,7 @@ h4,
 }
 
 p,
+label,
```

Comment:
@huong-li-nguyen does the `p` styling get used anywhere any more or is it redundant? What happens if we remove it? I only see it in some examples/tests now. I guess maybe it's what gets applied to `dcc.Markdown` also? Just curious...

Reply:
It is applied everywhere where we use `html.P` (e.g. the 404 error page) or even in our cards. In general, it's also better practice to have it in, especially for cases where people add their custom components and eventually have some paragraphs inside their components. Otherwise, they always have to use custom CSS to style any paragraphs. It is better if we ship this by default, but they can always overwrite it.

Reply:
Cool, that was my thinking also 👍